api/serverpb/json/header.json

```diff
@@ -17,6 +17,15 @@
         "description": "Returns the PMM Server logs.",
         "summary": "Logs",
         "operationId": "Logs",
+        "parameters": [
+          {
+            "type": "integer",
```

Comment:
can we replace it with a `boolean` type?

Reply:
@BupycHuk 
In JIRA ticket, we have: "User calls /logs.zip endpoint with `pprof=1` query parameter" that's why I used integer here. Shall I update JIRA ticket as well?

Reply:
boolean supports `pprof=1` as well and makes more sense

Reply:
Thank you for clarification. Changed.